src/Compilers/Core/Portable/Operations/ITryStatement.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeAnalysis.Semantics
     /// This interface is reserved for implementation by its associated APIs. We reserve the right to
     /// change it in the future.
     /// </remarks>
-    internal interface ITryStatement : IOperation // https://github.com/dotnet/roslyn/issues/22008
+    public interface ITryStatement : IOperation
```

Comment:
Consider naming `FinallyHandler` as `Finally` since we don't appear to use the term `Handler` for `Catches`.